### PR TITLE
Documented more common errors

### LINKED ISSUE #808: Document more common library update errors
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

### Describe your suggested feature

We should add to https://tachiyomi.org/help/guides/troubleshooting/#what-are-some-common-errors

...with more common things that we see nowadays such as:

- timeout
- connection reset
- SSL handshake aborted (e.g. https://cdn.discordapp.com/attachments/349436576037732355/924317756621455411/20211225_181042.jpg)
- Unable to resolve host (e.g. https://cdn.discordapp.com/attachments/349436576037732355/924265447996674118/IMG_20211225_121712.jpg)
- failed to connect to (e.g. https://cdn.discordapp.com/attachments/349436576037732355/924194749425020928/IMG_20211225_145916.jpg)
- not really an update error, but slow connections
- ...and I forget if there's anything else that commonly shows up in #support

We can link to this from the app to avoid some people going straight to Discord to ask what to do.

### Other details

_No response_

### PR DESCRIPTION
**Pull request by loocool2**
Closes #808

### CODE REVIEW

File: src/help/guides/troubleshooting.md [hunk 1]
```diff
@@ -8,46 +8,42 @@ lang: en-US
...
 :::
 ::: el-collapse-item title="HTTP error 403"
 Possible reasons for **HTTP error 403**:
 
-- The source you selected has [Cloudflare](#solving-the-cloudflare-issue) protection on, refer to the [Cloudflare](#solving-the-cloudflare-issue) guide to fix it.
+- The source you selected has [Cloudflare](#solving-cloudflare-issues) protection on, refer to the [Cloudflare](#solving-cloudflare-issues) guide to fix it.
 - The source is down, removed the manga, or banned your IP. Open <NavigationText item="webview"/> to check if this is the case.
 :::
 ::: el-collapse-item title="HTTP error 404"
 **HTTP error 404** probably means that the source is down or removed the manga. Open <NavigationText item="webview"/> to check if this is the case.
```
Comment (arkon) [inline src/help/guides/troubleshooting.md:24]: Should we prompt to migrate if desired?

File: src/help/guides/troubleshooting.md [hunk 2]
```diff
@@ -8,46 +8,42 @@ lang: en-US
...
+::: el-collapse-item title="Unable to resolve host / Connection failed / Failed to connect to / timeout / connection reset"
+Errors like **Unable to resolve host**, **Connection failed**, **Failed to connect to**, **timeout**, **connection reset** mean that something is preventing you from connecting to the source.
 
 Possible reasons include:
 - Your internet connection is not good enough to connect.
 - The app does not have access to the internet.
 - Your ISP has blocked the site.
 - The site is down.
-Try using different internet connection (switch to Wi-Fi, mobile data or a VPN). Try to enable <NavigationText item="more"/> → <NavigationText item="settings"/> → <NavigationText item="settings_advanced"/> → **DNS over HTTPS**.
+Try enabling <NavigationText item="more"/> → <NavigationText item="settings"/> → <NavigationText item="settings_advanced"/> → **DNS over HTTPS** first. Try using a different internet connection (switch to Wi-Fi, mobile data or a VPN) if **DNS over HTTPS** does not work.
```
Comment (arkon) [inline src/help/guides/troubleshooting.md:46]: Could we make this a list similar to the reasons? Another thing to try would also just be trying different networks or restarting your router.

File: src/help/guides/troubleshooting.md [hunk 3]
```diff
@@ -59,79 +55,44 @@ Try using different internet connection (switch to Wi-Fi, mobile data or a VPN).
...
 - The drive you're writing to is corrupted. Check using a file manager to see if it is accessible.
 :::
 ::: el-collapse-item title="Cannot Access SD Card"
-**Cannot Access SD Card** error is typically caused by having a file or folder name that is too long. Android's file manager does not support filenames longer than 255 characters. If you know the file or folder name that is the culprit, you can connect your SD card to your computer and shorten it. Otherwise, delete the Tachiyomi downloads folder off of the SD Card.
+The **Cannot Access SD Card** error can sometimes be fixed by changing the location of your downloads to somewhere else and back to the SD card, then restarting the app.
+
+Having a file or folder name that is too long can also cause this issue. Android's file manager does not support filenames longer than 255 characters. If you know the file or folder name that is the culprit, you can connect your SD card to your computer and shorten it. Otherwise, delete the Tachiyomi downloads folder off of the SD Card.
 :::
 ::: el-collapse-item title="Storage issues with Android 11"
```
Comment (arkon) [inline src/help/guides/troubleshooting.md:73]: Should probably say Android 11+ now.

File: src/help/guides/troubleshooting.md [hunk 4]
```diff
@@ -59,79 +55,44 @@ Try using different internet connection (switch to Wi-Fi, mobile data or a VPN).
...
-If you're using Preview, or a Fork, replace `eu.kanade.tachiyomi` with the other app's ID.
-  * **Tachiyomi Preview** → `eu.kanade.tachiyomi.debug`
-  * **TachiyomiAZ** → `eu.kanade.tachiyomi.az`
-  * **TachiyomiJ2K** → `eu.kanade.tachiyomi.j2k`
-  * **TachiyomiSY** → `eu.kanade.tachiyomi.sy`
-  * **Neko** → `tachiyomi.mangadex`
-
-***Please note that the command will not show any output if it is successful.***
-
+See [this](/help/faq/#android-11) section of the FAQ to learn how Scoped Storage affects Tachiyomi in Android 11 and how to fix it.
```
Comment (arkon) [inline src/help/guides/troubleshooting.md:74]: Same here.

File: src/help/guides/troubleshooting.md (hunk 1 above)
Comment (loocool2) [reply src/help/guides/troubleshooting.md:24]: sure

**Review by arkon — APPROVED** [review]

**Pull request merged**